quadratic/quadratic.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+/*Given coefficients A, B, and C of a quadratic equation Ax^2 + Bx + C = 0, write a program that prints the equation solutions x1 and x2.
+
+You will need to use math.Sqrt function to calculate a square root.
+
+Example:
+
+The program finds solutions of a quadratic equation.
+Enter the coefficients A, B and C:
+1 3 -10
+x1 = 2
+x2 = -5 */
+func main() {
+	fmt.Println("The program finds solutions of a quadratic equation.")
+	fmt.Println("Enter the coefficients A, B and C:")
+	var A float64
+	var B float64
+	var C float64
+	fmt.Scan(&A, &B, &C)
+	var D float64 = B*B - 4*A*C
+	if D > 0 {
```

Comment:
That's great you know branching (even though we haven't discussed it yet, and it wasn't required for this task).
Please either 
1) refactor the program to handle all possible cases correctly (including `a=0`), and please get rid of redundant condition computations.
2) or get rid of `if` statements (-;

Reply:
Fair remark. I'll try to simplify my code for myself tomorrow. I hope that after next lesson I could visit them online. And not with 144p  guess what has already been discussed and what hasn't. :)